__tests__/pages/index.test.tsx

```diff
@@ -1,24 +1,8 @@
-import React from "react";
-
-import { render, screen } from "@testing-library/react";
-import { latestNewsItemBuilder } from "~/lib/content/__mocks__/builders/informasi-terbaru";
-import HomePage, { getStaticProps } from "~/pages";
+import { getStaticProps } from "~/pages";
 
 jest.mock("~/lib/content/home-page");
 jest.mock("~/lib/content/welcome-message");
 
-describe("HomePage", () => {
-  it("renders the last updated time correctly", () => {
-    render(<HomePage latestNews={[latestNewsItemBuilder()]} />);
-
-    expect(
-      screen.getByText(
-        /Pembaruan terakhir pada Selasa, 27 Juli 2021 17.43 WIB/i,
-      ),
-    ).toBeVisible();
-  });
-});
-
```

Comment:
Thanks for working on it, @antoooks.
Hiwever, this test removal reduced the code coverage of the home page. Please put it back and change the assertion instead.

FYI, Mas @mazipan & @resir014.